lib/Environment.js

```diff
@@ -50,6 +52,28 @@ Environment.prototype = {
      */
     waitForPage: function (timeout) {
         throw new Error('You must redefine waitForPage method in child environment');
+    },
+
+    back: function () {
```

Comment:
For browser env it's just `window.history.back()`, for phantom it's `page.goBack()`. Phantom functions are more strong because they emulate real user actions.